syntax = "proto3";

message DigitalOutput{
    int32 numChannel =1;
    bool value=2;
}

message EpicEthernetOutput {

    repeated DigitalOutput digoutputs=1 ; //32 sorties [(nanopb).max_size = 32 , (nanopb).fixed_length = true]
    int32 nbChannel=2;
    string DeviceName=5; //
    string MacAddress=6; // 
}

